src/tests/ftest/util/data_mover_test_base.py

```diff
@@ -0,0 +1,398 @@
+#!/usr/bin/python
+"""
+(C) Copyright 2018-2020 Intel Corporation.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+GOVERNMENT LICENSE RIGHTS-OPEN SOURCE SOFTWARE
+The Government's rights to use, modify, reproduce, release, perform, display,
+or disclose this software are subject to the terms of the Apache License as
+provided in Contract No. B609815.
+Any reproduction of computer software, computer software documentation, or
+portions thereof marked with this legend must also reproduce the markings.
+"""
+from command_utils_base import CommandFailure
+from daos_utils import DaosCommand
+from test_utils_pool import TestPool
+from test_utils_container import TestContainer
+from ior_test_base import IorTestBase
+from data_mover_utils import DataMover
+from os.path import join
+import uuid
+
+class DataMoverTestBase(IorTestBase):
+    # pylint: disable=too-many-ancestors
+    """Base DataMover test class.
+
+    Sample Use Case:
+        set_ior_location_and_run("DAOS_UUID", "/testFile, pool1, cont1,
+                                 flags="-w -K")
+        set_src_location("DAOS_UUID", "/testFile", pool1, cont1)
+        set_dst_location("POSIX", "/some/posix/path/testFile")
+        set_ior_location_and_run("POSIX", "/some/posix/path/testFile",
+                                 flags="-r -R")
+    :avocado: recursive
+    """
+
+    def __init__(self, *args, **kwargs):
+        """Initialize a DataMoverTestBase object."""
+        super(DataMoverTestBase, self).__init__(*args, **kwargs)
+        self.dm_cmd = None
+        self.processes = None
+        self.pool = []
+        self.containers = []
+        self.uuids = []
```

Comment:
I was originally using `self.containers` in a different way, but now it's only used for generating a unique uns path.
I didn't think to change this. It's not necessary to store the containers, since I really just need a "counter" for the next uns path.
I can update this with the next iteration of datamover tests, since I also have some other refactoring changes to make.
Thank you!